src/test/java/uk/gov/pay/ledger/report/dao/PerformanceReportDaoIT.java

```diff
@@ -25,10 +27,16 @@
     @ClassRule
     public static AppWithPostgresAndSqsRule rule = new AppWithPostgresAndSqsRule();
 
-    private PerformanceReportDao transactionDao = new PerformanceReportDao(rule.getJdbi());
-
     private DatabaseTestHelper databaseTestHelper = aDatabaseTestHelper(rule.getJdbi());
 
+    private PerformanceReportDao transactionDao;
+
+    @Before
+    public void setUp() {
+        databaseTestHelper.truncateAllData();
+        transactionDao = new PerformanceReportDao(rule.getJdbi());
+    }
+
```

Comment:
if you're doing `databaseTestHelper.truncateAllData()` here then the method below `tearDown` could be got rid of

Reply:
Done